client/Dockerfile

```diff
@@ -4,10 +4,10 @@ WORKDIR /client
 
 COPY package*.json /client/
 
-RUN yarn
+RUN npm install
```

Comment:
Please revert this back to yarn and also on line 13 since. that is what create-react-app used

Reply:
Just saw that as well. This has been updated